src/ui/components/filters/geolocationcomponent.js

```diff
@@ -102,25 +102,12 @@ export default class GeoLocationComponent extends Component {
      * The query string to use for the input box, provided to template for rendering.
      * @type {string}
      */
-    this.query = this.core.globalStorage.getState(`${this.name}.${StorageKeys.QUERY}`) || '';
-    this.core.globalStorage.on('update', `${this.name}.${StorageKeys.QUERY}`, q => {
+    this.query = this.core.globalStorage.getState(`${StorageKeys.QUERY}.${this.name}`) || '';
+    this.core.globalStorage.on('update', `${StorageKeys.QUERY}.${this.name}`, q => {
       this.query = q;
       this.setState();
     });
 
-    /**
-     * The filter node to use for the current query
-     * @type {FilterNode}
-     */
-    let filterNode = this.core.globalStorage.getState(`${this.name}.${StorageKeys.STATIC_FILTER_NODE}`) || {};
-    if (typeof filterNode === 'string') {
-      try {
-        filterNode = JSON.parse(filterNode);
-      } catch (e) {}
-    }
-
-    this.filterNode = FilterNodeFactory.from({ ...filterNode });
```

Comment:
Was this just unused?

Reply:
yeah, I was trying to match the old behavior 1 for 1, but if originalFilter wasn't doesn't doing anything this wasn't doing anything